run_bot_rabbitmq.sh

```
@@ -0,0 +1,163 @@
+        print_info "Building projects..."
+        nix-shell --run "npm run build --prefix recording_server && npm run build-dev --prefix recording_server/chrome_extension"
+    fi
+    
+    print_success "Environment ready"
+}
+
+# Load environment variables
+load_env() {
+    if [ -f ".env" ]; then
+        print_info "Loading environment variables from .env..."
+        source .env
```

Shellcheck (0.10.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)